Code/Editor/Platform/Linux/Editor/Core/QtEditorApplication_linux.cpp

```diff
@@ -23,16 +25,33 @@ namespace Editor
         return nullptr;
     }
 
+    xcb_connection_t* EditorQtApplicationXcb::GetXcbConnectionFromQt()
+    {
+        QPlatformNativeInterface* native = platformNativeInterface();
+        if (!native)
```

Comment:
Should put an AZ_Assert on native here unless `nullptr` is something that is handled by the caller

Reply:
Done.